shared/js/background/chrome-wrapper.es6.js

```diff
@@ -20,6 +20,11 @@ const getFromStorage = (key, cb) => {
         cb(result[key])
     })
 }
+const getFromManagedStorage = (keys, cb) => {
+    chrome.storage.managed.get(keys, (result) => {
+        cb(result)
```

Comment:
```suggestion
        if (!browser.runtime.lastError) {
            cb(result)
        } else {
            console.warn('Managed storage not available.', browser.runtime.lastError)
        }
```

We need to check for error because if there is no policy manifest available Firefox is throwing this error:

> Unchecked lastError value: Error: Managed storage manifest not found